src/Footer.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Footer = (props) => {
```

Comment:
Have you checked if this component is working as expected? If not could it be due to the syntax error on the top?